org.metaborg.meta.nabl2.java/src/main/java/org/metaborg/meta/nabl2/solver/components/SymbolicSolver.java

```diff
@@ -31,7 +31,7 @@ public SymbolicSolver(Solver solver) {
     }
 
     @Override protected Unit doAdd(ISymbolicConstraint constraint) throws UnsatisfiableException {
-        constraint.matchOrThrow(CheckedCases.of(fact -> {
+        constraint.<Unit, UnsatisfiableException>matchOrThrow(CheckedCases.of(fact -> {
```

Comment:
I don't know why, but I had to add these types to get the project to compile at all.

Reply:
Inference does not always work as you'd want. I had other places where I had to make them explicit as well.